torch/_inductor/fx_passes/post_grad.py

```diff
@@ -95,7 +95,7 @@ def post_grad_passes(gm: torch.fx.GraphModule, is_inference: bool):
 
     stable_topological_sort(gm.graph)
 
-    move_constructors_to_cuda(gm.graph)
+    move_constructors_to_cuda(gm)
```

Comment:
Can we switch this back to taking in a graph ? We don't want to do a gm recompile here, and I think taking in a graph instead of a gm makes it more clear that this pass does not recompile.

Reply:
Yes. All FX passes take in a GraphModule.

Reply:
But, since we are not moving this pass to torch/fx/passes, we can simply not inherit from `PassBase`, and take in a graph.